build/vsts-ci.yml

```diff
@@ -6,7 +6,7 @@ resources:
   containers:
   - container: CentosContainer
     image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-mlnet-8bba86b-20190314145033
-
+g
```

Comment:
what is this "g"?

Reply:
a test for the reviewers 😆

Reply:
Must have been an accidental keyboard hit. Good catch.